src/managers/notification-manager.ts

```diff
@@ -63,7 +62,6 @@ class NotificationManager extends LitElement {
     return html`
       <ha-toast
         leading
-        dir=${computeRTL(this.hass) ? "rtl" : "ltr"}
```

Comment:
Hi @wendevlin @bramkragten why did you remove this?

Reply:
As far as I remember we found it that it had no function.

Reply:
@wendevlin I simulated a toast now and it looks fine. The direction is inherited properly so you're right. Thanks.

Reply:
thanks for testing :slightly_smiling_face: